### DISCUSSION TITLE
Beware that it eats up a lot of RAM

### DISCUSSION THREAD

**Discussion by NoMansPC** [lllyasviel/Fooocus#21] [General]
I'm guessing the recommended minimum to run this is way over 16gb of RAM because it completely froze up my PC. I thought it would run well because in the read me it says it needs only 4gb of VRAM, but I think it compensates that by expecting you to have way more RAM. It wasn't even using my 2060 to its maximum while generating the images since the fans weren't even spinning fast enough to make noise.

**Comment by bccafe66** [comment]
Using "nvitop.exe", reading 6499MIB/8192MIB .

**Reply by aimarxjg** [reply]
Excuse me, where is this document?"nvitop.exe",

**Comment by lllyasviel** [comment]
the real RAM consumption on low end device is lower than high-end devices with lots of RAM
see also updated readme installation guides with images and explanations

**Comment by bccafe66** [comment]
Excellent Utilities: nvitop – GPU process management
Excellent Utilities: nvitop - GPU process management - Page 2 of 2 -
LinuxLinks <https://www.linuxlinks.com/excellent-utilities-nvitop/2/>
